Documentation/Installation/TutorialDdev.rst

```diff
@@ -0,0 +1,196 @@
+
+=============================================
+Tutorial "Installing TYPO3 v11 LTS with DDEV"
+=============================================
+
+We show you step-by-step how to install TYPO3 on a local machine utilizing
+DDEV, Docker and Composer.
+
+..  youtube:: HW7J3G1SqZw
+
+.. contents::
+   :local:
+
+Preparations to install DDev
+============================
+
+Do the following steps if DDEV is not installed on your system yet:
+
+..  rst-class:: bignums-xxl
+
+#.  Install Docker
+
+    Go to `the Docker website <https://www.docker.com/>`__ and install the
+    version of Docker recommended for your operation system.
+
+#.  Install DDEV
+
+    Follow the steps at `Get Started with
+    DDEV <https://ddev.readthedocs.io/en/stable/>`__ for your operation
+    system.
+
+..  hint::
+    If you need help installing DDEV, go to the
+    `DDEV Discord server <https://discord.gg/kDvSFBSZfs>`__.
+
+Install TYPO3 in DDEV
+=====================
+
+..  rst-class:: bignums-xxl
+
+#.  Create a directory to install your TYPO3 in
+
+    ..  code-block:: bash
+
+        mkdir t3example
+        cd t3example
+
+#.  Create a preconfigured DDEV project
+
+    .. rst-class:: bignums-warning
+
+    a.  By answering the prompt (easier to do by heart):
+
+        ..  code-block:: bash
+
+            ddev config
+
+            # Give the following answers when prompted:
+
+            Project name (t3example):
+
+            Docroot Location (current directory):public
+
+            Create docroot at /home/myuser/projects/t3/t3example/public? [Y/n] (yes): Y
+
+            Project Type [php, typo3, ...] (php): typo3
+
+    b.  Or by supplying the parameters (easier to do automatically):
+
+        ..  code-block:: bash
+
+            ddev config  --project-type=typo3 --docroot=public --create-docroot
+
+    project-type
+        Should always be "typo3"
+
+    docroot
+        Is the folder in which all files that have to be reached by
+        the browser. This folder is commonly called :file:`public`.
+
+    create-docroot
+        As the directory does not exist yet allow DDEV to create it for you.
+
+#.  Start the project
+
+    ..  code-block:: bash
+
+        ddev start
+
+    Now the webserver is running but TYPO3 is not installed yet.
+
+#.  Install a basic installation of TYPO3
+
+    ..  code-block:: bash
+
+        ddev composer create "typo3/cms-base-distribution:^11"
+
+    As we just created the project and have no data there yet, answer yes
+    when prompted if it is ok to overwrite most files in this directory.
+
+    ..  note::
+        You now have a **Composer-based TYPO3 installation**. In case anyone
+        asks you.
+
+#.  Run the install tool setup
+
+    .. rst-class:: bignums-warning
+
+    a.  You can do this from console:
+
+        ..  code-block:: bash
+
+            ddev typo3cms install:setup
+
+        When prompted enter a username and password for you main admin user
+        (who is also system maintainer).
```

Comment:
Maybe link "system maintainer", not everyone may know, what that is: https://docs.typo3.org/m/typo3/tutorial-getting-started/11.5/en-us/UserManagement/BackendPrivileges/Index.html#system-maintainers